pkg/utils/apiresources_test.go

```diff
@@ -305,6 +305,24 @@ func TestDisabledResourceConfigMixedParse(t *testing.T) {
 			}
 		}
 	}
+	for _, test := range tests {
+		// test allow list
+		r := NewResourceConfig(true)
+		if err := r.Parse(test.input); err != nil {
+			t.Fatalf("Unexpected error: %v", err)
```

Comment:
```suggestion
			t.Fatalf("Parse() returned error: %v", err)
```

https://go.dev/wiki/TestComments#identify-the-function
`In most tests, failure messages should include the name of the function that failed, even though it seems obvious from the name of the test function.`